src/realtime-callbacks.js

```diff
@@ -56,6 +56,80 @@ module.exports.setNow = function(f) {
 };
 let _now = Date.now;
 
+// Internal hash of active debouncers
+const _debouncers = {};
+
+class Debouncer {
+
+    constructor(jobName) {
+        this.jobName = jobName;
+    }
+
+    complete() {
+        delete _debouncers[this.jobName];
+        this.func();
```

Comment:
You're assuming that a `func` is always provided, even though you're null guarding on `start(func, waitMs)`. Which is it?